react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import { makeStyles } from "@material-ui/core";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import DeckGLMap from "../../DeckGLMap";
+
+export default {
+    component: DeckGLMap,
+    title: "DeckGLMap / Lasso Layer",
+} as ComponentMeta<typeof DeckGLMap>;
+
+const useStyles = makeStyles({
+    main: {
+        height: 500,
+        border: "1px solid black",
+        position: "relative",
+    },
+    legend: {
+        width: 100,
+        position: "absolute",
+        top: "0",
+        right: "0",
+    },
+});
+
+export const lassoLayerTemplate: ComponentStory<typeof DeckGLMap> = (args) => {
+    const [editedData, setEditedData] = React.useState(args.editedData);
+    const [isLassoSelectionAvailable, setIsLassoSelectionAvailable] =
+        React.useState<boolean>(false);
+    const handleChange1 = () => {
+        setIsLassoSelectionAvailable(true);
+    };
+
+    const handleChange2 = () => {
+        setIsLassoSelectionAvailable(false);
+    };
+    React.useEffect(() => {
+        setEditedData(args.editedData);
+    }, [args.editedData]);
+
+    return (
+        <>
+            <div className={useStyles().main}>
+                <DeckGLMap
+                    {...args}
+                    editedData={editedData}
+                    setProps={(updatedProps) => {
+                        setEditedData(updatedProps);
+                    }}
+                    isLassoSelectionAvailable={isLassoSelectionAvailable}
+                />
+            </div>
+            <button onClick={handleChange1}> Enable Lasso Selection </button>
+            <button onClick={handleChange2}> Disable Lasso Selection </button>
```

Comment:
Since this is a toggle, we could represent it as checkbox or a toggle button instead of two buttons, with the benefit of showing the current state.